docs/api/synthetics/monitors/update-monitor-api.asciidoc

```diff
@@ -0,0 +1,271 @@
+  
+    * `send` (Optional, string): check.send: 'Hello World'
+    * `receive` (Optional, string): check.receive: 'Hello World'
+
+`proxy_url`:: (Optional, string): The URL of the SOCKS5 proxy to use when connecting to the server. The value must be a URL with a scheme of `socks5://`. If the SOCKS5 proxy server requires client authentication, then a username and password can be embedded in the URL. When using a proxy, hostnames are resolved on the proxy server instead of on the client. You can change this behavior by setting the `proxy_use_local_resolver` option.
+
+`proxy_use_local_resolver`:: (Optional, boolean, default: false): A Boolean value that determines whether hostnames are resolved locally instead of being resolved on the proxy server. The default value is false, which means that name resolution occurs on the proxy server.
+
+*ICMP Monitor Fields*:
+
+`host`:: (Required, string): Host to ping.
+`wait`:: (Optional, number, default: 1): Wait time in seconds.
+
+*Browser Monitor Fields*:
+
+`inline_script`:: (Required, string): The inline script.
+
+`screenshots`:: (Optional, string, default: "on"): Screenshot option, either "on",  "off" or "only-on-failure".
+
+`synthetics_args`:: (Optional, array): Synthetics agent CLI arguments.
+
+`ignore_https_errors`:: (Optional, boolean, default: false): Whether to ignore HTTPS errors.
+
+`playwright_options`:: (Optional, object): Playwright options.
+
+==== Examples
+
+Here are some examples of creating monitors of different types:
+
+**HTTP Monitor**:
+Create an HTTP monitor to check a website's availability.
+
+[source,sh]
+--------------------------------------------------
+PUT /api/synthetics/monitors/<monitor_id>
+{
+  "type": "http",
+  "name": "Website Availability",
+  "url": "https://example.com",
+  "tags": ["website", "availability"],
+  "locations": ["united_kingdom"]
+}
+--------------------------------------------------
+
+**TCP Monitor**:
+Create a TCP monitor to monitor a server's availability.
+
+[source,sh]
+--------------------------------------------------
+PUT /api/synthetics/monitors/<monitor_id>
+{
+  "type": "tcp",
+  "name": "Server Availability",
+  "host": "example.com",
+  "private_locations": ["my_private_location"]
+}
+--------------------------------------------------
+
+**ICMP Monitor**:
+Create an ICMP monitor to perform ping checks.
+
+[source,sh]
+--------------------------------------------------
+PUT /api/synthetics/monitors/<monitor_id>
+{
+  "type": "icmp",
+  "name": "Ping Test",
+  "host": "example.com",
+  "locations": ["united_kingdom"]
+}
+--------------------------------------------------
+
+**Browser Monitor**:
+Create a Browser monitor to check a website.
+
+[source,sh]
+--------------------------------------------------
+PUT /api/synthetics/monitors/<monitor_id>
+{
+  "type": "browser",
+  "name": "Example journey",
+  "inline_script": "step('Go to https://google.com.co', () => page.goto('https://www.google.com'))",
```

Comment:
```suggestion
```

We state that the inline script cannot be edited right below this example, but in the example we show the user posting an inline script update.

Reply:
Ah my mistake then